content/faq.md

```diff
@@ -109,29 +161,55 @@ You can also inspect and execute functions in the [azimuth.eth](https://ethersca
 
 ### What are comets and moons? {#comets-moons}
 
-In addition to the three classes of identities mentioned above, there are two other kinds of Urbit ships that are _not_ registered in the Urbit ID / Azimuth contracts.
+In addition to the three classes of identities mentioned above, there are two other kinds of Urbit [ship](@/docs/glossary/ship.md)s that are _not_ registered in the Urbit ID / Azimuth contracts.
 
-Moons are 64 bits, issued by planets, and have names like `~doznec-salfun-naptul-habrys`. Moons are meant for connected devices: phones, desktops, smart TVs, digital thermostats, and other IoT devices. Moons are subordinated to their parent planet.
+[Moons](@/docs/glossary/moon.md) are 64 bits, issued by
+[planets](@/docs/glossary/planet.md), and have names like
+`~doznec-salfun-naptul-habrys`. Moons are meant for connected devices: phones,
+desktops, smart TVs, digital thermostats, and other IoT devices. Moons are
+subordinate to their parent planet. A strong analogy here is that ordinary
+social media sites are somewhat like planets, and your account on them is a
+moon. Urbit elevates everybody to the level of planet.
 
-Comets are 128 bits and have no parents. They can be launched by anyone. They are temporary, disposable identities. Being disposable and essentially unlimited, they will likely not be trusted by default by others on the Urbit OS network, though you shouldn't have any problem until the network grows much larger. They have long, hard-to-memorize names, like `~racmus-mollen-fallyt-linpex-watres-sibbur-modlux-rinmex`.
+[Comets](@/docs/glossary/comet.md) are 128 bits and have no parents. They can be launched by anyone. They are temporary, disposable identities. Being disposable and essentially unlimited, they will likely not be trusted by default by others on the [Urbit OS](#what-is-arvo) network, though you shouldn't have any problem until the network grows much larger. They have long, hard-to-memorize names, like `~racmus-mollen-fallyt-linpex-watres-sibbur-modlux-rinmex`.
 
 ### What is a `@p`? Why is my username generated for me? {#what-is-patp}
 
-A `@p` (pronounced *pat-pee*) is a name like `~zod` or `~lodleb-ritrul` composed of pronounceable, three-letter phonemic elements like `zod`, `lod`, `leb`, `rit` and `rul`. Shorter names, such as `~zod` and `~marzod`, are assigned to ships with special duties on the Arvo network stars and galaxies, respectively. Longer names like `~palfun-foslup` are identities for typical users.
+A `@p` (pronounced *pat-pee*) is a name like `~zod` or `~lodleb-ritrul` composed
```

Comment:
For laughs:

```suggestion
A `@p` ([pronounced](@/docs/tutorials/hoon/hoon-school/hoon-syntax/#reading-hoon-aloud) *pat-pee*) is a name like `~zod` or `~lodleb-ritrul` composed
```

Reply:
That relative link will crash the build — you want `@/docs/tutorials/hoon/hoon-school/hoon-syntax.md#reading-hoon-aloud`.

Reply:
https://github.com/urbit/urbit.org/pull/538/commits/e784103b041dbe6f246df675f4039ba759235526